src/Lib/xprs.jl

```diff
@@ -12,10 +12,8 @@
 """
 const XPRSobject = Ptr{Cvoid}
 
-# Struct does not exist in v33
```

Comment:
Can we revert these please? I can find a copy of xprs33 if we need

Reply:
Oh, i was wondering what was the right way to do this.
Yeah, i think it is the best choice, just creating a new folder for the new version is way less confusing for support.